virtual-desktop/src/app/plugin-manager/plugin-factory/iframe/iframe-plugin.component.ts

```diff
@@ -56,6 +56,25 @@ export class IFramePluginComponent {
     if(split[0] === 'registerAdapterInstance' && this.instanceId == -1){
       this.instanceId = data.request.instanceId;
       this.frameSource = message.source;
+      try{
+        this.frameSource.postMessage({
+          key: -1,
+          constructorData: {
+            pluginDef: JSON.parse(JSON.stringify(this.pluginDefintion)),
+            launchMetadata: this.launchMetadata
+          },
+          instanceId: this.instanceId
+        }, '*')
+      }catch(e){
+        this.frameSource.postMessage({
```

Comment:
This must have a log message, so that we can spot if this happens, because it should never happen.

Reply:
@1000TurquoisePogs  There's a log message a few lines down, but because this error is caught, I don't think it will get outputted to the console, so it would be good to add that 'e' in